modules/test/playwright/tests/workspaces/liferay-workspace-partner/pages/MDFRequestFormPage/MDFRequestFormGoals.ts

```diff
@@ -0,0 +1,210 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page, expect} from '@playwright/test';
+
+import {
+	MDFRequestAdditionalOptions,
+	MDFRequestLiferayBusinessSalesGoals,
+	MDFRequestTargetAudienceRoles,
+	MDFRequestTargetMarkets,
+} from '../../utils/enums';
+
+export type MDFRequestFormGoalsContent = {
+	additionalOptions?: MDFRequestAdditionalOptions[];
+	companyName: string;
+	liferayBusinessSalesGoals: MDFRequestLiferayBusinessSalesGoals[];
+	liferayBusinessSalesGoalsOther?: string;
+	overallCampaignDescription: string;
+	overallCampaignName: string;
+	targetAudienceRoles: MDFRequestTargetAudienceRoles[];
+	targetMarkets: MDFRequestTargetMarkets[];
+};
+
+export class MDFRequestFormGoals {
+	readonly additionalOptions: {
+		dxpMigrationUpgrade: Locator;
+		migrationFromCompetitorPlatform: Locator;
+	};
+	readonly companyName: Locator;
+	readonly liferayBusinessSalesGoals: {
+		leadGeneration: Locator;
+		nurtureExistingProspects: Locator;
+		other: Locator;
+		thoughtLeadership: Locator;
+	};
+	readonly liferayBusinessSalesGoalsOther: Locator;
+	readonly overallCampaignDescription: Locator;
+	readonly overallCampaignName: Locator;
+	readonly page: Page;
+	readonly targetAudienceRoles: {
+		administrator: Locator;
+		associateAnalyst: Locator;
+		cLevelExecutiveVP: Locator;
+		developerEngineer: Locator;
+		directorManager: Locator;
+		ecommerceLeadership: Locator;
+		independentContractor: Locator;
+		projectManager: Locator;
+	};
+	readonly targetMarkets: {
+		aerospaceDefense: Locator;
+		agriculture: Locator;
+		automotive: Locator;
+		constructionEngineering: Locator;
+		consultingMarketResearch: Locator;
+		education: Locator;
+		energy: Locator;
+		financialServices: Locator;
+		foodServices: Locator;
+		governmentFederal: Locator;
+		governmentStateLocal: Locator;
+		healthcare: Locator;
+		hospitalityLeisure: Locator;
+		insurance: Locator;
+		manufacturing: Locator;
+		mediaEntertainment: Locator;
+		notForProfitNGO: Locator;
+		pharmaceuticals: Locator;
+		professionalServicesAgencyBusiness: Locator;
+		professionalServicesTechnicalWebIT: Locator;
+		retailConsumerProducts: Locator;
+		technology: Locator;
+		telecommunications: Locator;
+		transportation: Locator;
+		utilities: Locator;
+		wholesaleDistribution: Locator;
+	};
+
+	constructor(page: Page) {
+		this.companyName = page.locator('select[name="company"]');
```

Comment:
I think we should sort these in alphabetical order rather than form order. Alpha order will be easier to maintain in the long run, as well as be consistent among the whole project if everything is in alpha order where possible.

Reply:
Makes sense